test/unit/account_info/test_account_info.py

```diff
@@ -414,30 +415,24 @@ def _make_sqlite_account_info(self, env=None, last_upgrade_to_run=None):
                 last_upgrade_to_run=last_upgrade_to_run,
             )
 
-    def test_uses_default(self):
-        account_info = self._make_sqlite_account_info(
-            env={
-                'HOME': self.test_home,
-                'USERPROFILE': self.test_home,
-            }
-        )
-        actual_path = os.path.abspath(account_info.filename)
-        assert os.path.join(self.test_home, '.b2_account_info') == actual_path
-
     def test_uses_xdg_config_home(self, apiver):
+        is_xdg_os = bool(SqliteAccountInfo.get_xdg_config_path())
```

Comment:
this worries me as, as we don't verify `get_xdg_config_path()`  properly recognizing XDG compatibility at any point

maybe instead of this please use your variation of platform xdg recognition here?
OR the one from `platformdirs` as test-only dependency?

Reply:
adding another test for that